source/puppet-agent/_release_notes.md

```diff
@@ -1,10 +1,13 @@
 ### Release Notes
 
+* **1.2.7**, Oct. 29, 2015: Adds or improves support for AIX 5.3, [6.1](https://tickets.puppetlabs.com/browse/PA-32), and [7.1](https://tickets.puppetlabs.com/browse/PA-33) on PowerPC, Solaris 11 on [SPARC](https://tickets.puppetlabs.com/browse/PA-4) and [i386](https://tickets.puppetlabs.com/browse/PA-5), and Enterprise Linux 4. Updates Ruby, [Facter](/facter/3.1/release_notes.html#facter-311), and [Hiera](/hiera/latest/release_notes.html#hiera-304).
```

Comment:
Could you link AIX 5.3 to https://tickets.puppetlabs.com/browse/PA-41 (just moved the RE ticket to PA like the others)